packages/now-python/test/fixtures/10-url-params-http-handler/now.json

```diff
@@ -3,7 +3,7 @@
   "builds": [{ "src": "*.py", "use": "@now/python" }],
   "routes": [{ "src": "/another", "dest": "custom.py" }],
   "probes": [
-    { "path": "/?hello=/", "mustContain": "path=/?hello=/" },
-    { "path": "/another?hello=/", "mustContain": "path=/another?hello=/" }
+    { "path": "/?hello=/", "mustContain": "path=/?hello=%2F" },
+    { "path": "/another?hello=/", "mustContain": "path=/another?hello=%2F" }
```

Comment:
Please don't change this test.

You introduced a bug with your PR, this test is fine.

Reply:
sorry. just added what you commented here: 

https://spectrum.chat/zeit/now/double-url-encoding~1f95aed5-b226-4976-8681-47e5608be8d7?m=MTU3MTc0ODk5OTUyOA==

Reply:
theres an error in URLs with space in params.